### PR TITLE
Ticket8427: migrate all motor settings from instrument config directories

### PR DESCRIPTION
**Pull request by Tom-Willemsen**
https://github.com/ISISComputingGroup/IBEX/issues/8427

sorry

### CODE REVIEW

File: settings/OSIRIS/galil/motorExtensions.cmd [hunk 1]
```diff
@@ -0,0 +1,5 @@
+############ Motor Extensions ##############
+# Create a soft motor record on top of a normal motor record
+# Copy this file to C:\Instrument\Settings\config\<machine>\configurations\galil
+
+# < $(GALILCONFIG)/zoom_bumpstrip.cmd
```
Comment (Tom-Willemsen) [inline settings/OSIRIS/galil/motorExtensions.cmd:5]: hmm

File: settings/PEARL_SETUP/linmot/linmot01.cmd [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+LinmotCreateAxis("Linmot","C",0,"",1)
+LinmotCreateAxis("Linmot","D",0,"",1)
+LinmotCreateAxis("Linmot","E",0,"",1)
+LinmotCreateAxis("Linmot","F",0,"",1)
+LinmotCreateAxis("Linmot","G",0,"",1)
+LinmotCreateAxis("Linmot","H",0,"",1)
+
+ 
+
+epicsEnvSet("HOME_MODE_1" "$(Linmot)/gmc/Linmot_Home_Dummy_Do_Nothing.gmc")
```
Comment (Tom-Willemsen) [inline settings/PEARL_SETUP/linmot/linmot01.cmd:30]: This seems very very unlikely, passing galil code to a linmot!?

File: settings/POLARIS/galil/jaws.cmd [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+$(IFIOC_GALIL_01) dbLoadRecords("$(JAWS)/db/jaws.db","P=$(MYPVPREFIX)MOT:,JAWS=JAWS1:,mXN=MTR0104,mXS=MTR0103,mXW=MTR0102,mXE=MTR0101")
+$(IFIOC_GALIL_01) dbLoadRecords("$(JAWS)/db/jaws.db","P=$(MYPVPREFIX)MOT:,JAWS=JAWS2:,mXN=MTR0108,mXS=MTR0107,mXW=MTR0106,mXE=MTR0105")
+$(IFIOC_GALIL_02) dbLoadRecords("$(JAWS)/db/jaws.db","P=$(MYPVPREFIX)MOT:,JAWS=JAWS3:,mXN=MTR0204,mXS=MTR0203,mXW=MTR0202,mXE=MTR0201")
+$(IFIOC_GALIL_02) dbLoadRecords("$(JAWS)/db/jaws.db","P=$(MYPVPREFIX)MOT:,JAWS=JAWS4:,mXN=MTR0208,mXS=MTR0207,mXW=MTR0206,mXE=MTR0205")
+$(IFIOC_GALIL_03) dbLoadRecords("$(JAWS)/db/jaws.db","P=$(MYPVPREFIX)MOT:,JAWS=JAWS5:,mXN=MTR0301,mXS=MTR0303,mXW=MTR0304,mXE=MTR0302,ASG=MANAGER")
+
+# Define jaws as soft motor records using the second galil controller, if present
+
+$(IFIOC_GALIL_01) dbLoadRecords("C:/Instrument/Apps/EPICS/support/motorExtensions/master/db/polarisjawset.db", "P=$(MYPVPREFIX), COL_HEIGHT=79.4, COL_WIDTH=82.2, MOD_TO_SAMPLE=14000, MOD_TO_COL=1610, MOD_TO_JAWS1=6502, MOD_TO_JAWS2=9440, MOD_TO_JAWS3=11085, MOD_TO_JAWS4=11735, MOD_TO_JAWS5=13175")
```
Comment (Tom-Willemsen) [inline settings/POLARIS/galil/jaws.cmd:13]: Technically won't break but should be a macro

File: settings/POLREF/galil/analyser.bat [hunk 4]
```diff
@@ -0,0 +1,52 @@
+caput IN:POLREF:MOT:MTR0508.RMOD 0
+caput IN:POLREF:MOT:MTR0508_HOMEVAL_SP 0
+caput IN:POLREF:MOT:MTR0508_WLP_CMD  0
+caput IN:POLREF:MOT:MTR0508_OFFDELAY_SP 2
+caput IN:POLREF:MOT:MTR0508_ONDELAY_SP 0
+caput IN:POLREF:MOT:MTR0508_JAH_CMD No
+						  8
+						  8
```
Comment (Tom-Willemsen) [inline settings/POLREF/galil/analyser.bat:8]: pretty sure this isn't valid `.bat` syntax...

File: settings/SANS2D/SMC100_01/motionsetpoints.cmd [hunk 5]
```diff
@@ -0,0 +1,7 @@
+
+# Setting up the Sample Changer values
+$(IFIOC_SMC100_01) epicsEnvSet "LOOKUPFILE3" "$(ICPCONFIGROOT)/motionSetPoints/dls_sample_changer.txt"
+$(IFIOC_SMC100_01) motionSetPointsConfigure("LOOKUPFILE3","LOOKUPFILE3", 2)
+$(IFIOC_SMC100_01) dbLoadRecords("$(MOTIONSETPOINTS)/db/motionSetPointsDoubleAxis.db","P=$(MYPVPREFIX)LKUP:DLS:,AXIS0=$(MYPVPREFIX)MOT:MTR1501,AXIS1=$(MYPVPREFIX)MOT:SAMP:Y,TOL=1,LOOKUP=LOOKUPFILE3")
+        
+$(IFIOC_SMC100_01) dbLoadRecordsLoop("$(MOTIONSETPOINTS)/db/inPos.db","P=$(MYPVPREFIX)LKUP:DLS:,AXIS0=$(MYPVPREFIX)MOT:MTR1501,AXIS1=$(MYPVPREFIX)MOT:SAMP:Y,TOL=1,LOOKUP=LOOKUPFILE3","NUMPOS", 0, 5)
```
Comment (Tom-Willemsen) [inline settings/SANS2D/SMC100_01/motionsetpoints.cmd:7]: eof newline?

File: settings/SANS2D/galil/motionsetpoints.cmd [hunk 6]
```diff
@@ -0,0 +1,7 @@
+
+# Setting up the Sample Changer values
+$(IFIOC_GALIL_03) epicsEnvSet "LOOKUPFILE2" "$(ICPCONFIGROOT)/motionSetPoints/sample.txt"
+$(IFIOC_GALIL_03) motionSetPointsConfigure("LOOKUPFILE2","LOOKUPFILE2", 2)
+$(IFIOC_GALIL_03) dbLoadRecords("$(MOTIONSETPOINTS)/db/motionSetPointsDoubleAxis.db","P=$(MYPVPREFIX)LKUP:SAMPLE:,AXIS0=$(MYPVPREFIX)MOT:SAMP:X,AXIS1=$(MYPVPREFIX)MOT:SAMP:Y,TOL=1,LOOKUP=LOOKUPFILE2")
+
+$(IFIOC_GALIL_03) dbLoadRecordsLoop("$(MOTIONSETPOINTS)/db/inPos.db","P=$(MYPVPREFIX)LKUP:SAMPLE:,AXIS0=$(MYPVPREFIX)MOT:SAMP:X,AXIS1=$(MYPVPREFIX)MOT:SAMP:Y,TOL=1,LOOKUP=LOOKUPFILE2","NUMPOS", 0, 30)
```
Comment (Tom-Willemsen) [inline settings/SANS2D/galil/motionsetpoints.cmd:7]: eof newline

File: settings/SANS2D/galilmul/galilmul04.cmd [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+GalilCreateAxis("Galil1","B",1,"",1)
+GalilCreateAxis("Galil1","C",1,"",1)
+GalilCreateAxis("Galil1","D",1,"",1)
+GalilCreateAxis("Galil1","E",1,"",1)
+GalilCreateAxis("Galil1","F",1,"",1)
+GalilCreateAxis("Galil1","G",1,"",1)
+GalilCreateAxis("Galil1","H",1,"",1)
+
+epicsEnvSet "HOME_HEADER" "$(GALIL)/gmc/galil_Default_Header.gmc"
+epicsEnvSet "HOME_HEADER_2" "$(ICPCONFIGROOT)/galil/galil_Home_sans2d_rear_detector.gmc"
```
Comment (Tom-Willemsen) [inline settings/SANS2D/galilmul/galilmul04.cmd:13]: FIX THIS this will be wrong after migration

File: settings/SELAB/galil/galil01.cmd [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+GalilCreateAxis("Galil","A",1,"",1)
+GalilCreateAxis("Galil","B",1,"",1)
+GalilCreateAxis("Galil","C",1,"",1)
+GalilCreateAxis("Galil","D",1,"",1)
+GalilCreateAxis("Galil","E",1,"",1)
+GalilCreateAxis("Galil","F",1,"",1)
+GalilCreateAxis("Galil","G",1,"",1)
+GalilCreateAxis("Galil","H",1,"",1)
+
+<<<<<<< HEAD
```
Comment (Tom-Willemsen) [inline settings/SELAB/galil/galil01.cmd:12]: Not ideal

File: settings/SURF/galil/jaws.cmd [hunk 9]
```diff
@@ -0,0 +1,9 @@
+############ GEM jaws ##############
```
Comment (Tom-Willemsen) [inline settings/SURF/galil/jaws.cmd:1]: hmm

File: settings/SURF/twincat/axes.cmd [hunk 10]
```diff
@@ -0,0 +1,2 @@
+#TODO
```
Comment (Tom-Willemsen) [inline settings/SURF/twincat/axes.cmd:1]: hmm

File: settings/POLREF/galil/galil01.cmd [hunk 11]
```diff
@@ -0,0 +1,44 @@
...
+epicsEnvSet "HOME_MODE_1" "$(GALIL)/gmc/galil_Home_Dummy_Do_Nothing.gmc"
+epicsEnvSet "HOME_MODE_2" "$(GALIL)/gmc/galil_Home_Dummy_Do_Nothing.gmc"
+epicsEnvSet "HOME_MODE_3" "$(GALIL)/gmc/galil_Home_Dummy_Do_Nothing.gmc"
+epicsEnvSet "HOME_MODE_4" "$(GALIL)/gmc/galil_Home_Dummy_Do_Nothing.gmc"
+epicsEnvSet "HOME_MODE_5" "$(GALIL)/gmc/galil_Home_Dummy_Do_Nothing.gmc"
+epicsEnvSet "HOME_MODE_6" "$(GALIL)/gmc/galil_Home_Dummy_Do_Nothing.gmc"
+epicsEnvSet "HOME_MODE_7" "$(GALIL)/gmc/galil_Home_Dummy_Do_Nothing.gmc"
+epicsEnvSet "HOME_MODE_8" "$(GALIL)/gmc/galil_Home_Dummy_Do_Nothing.gmc"
+epicsEnvSet "HOME_FOOTER" "$(GALIL)/gmc/galil_Default_Footer.gmc"
+epicsEnvSet "POLREF_SPECIAL" "C:/Instrument/Settings/config/NDXPOLREF/configurations/galil/combined_header.gmc"
```
Comment (Tom-Willemsen) [inline settings/POLREF/galil/galil01.cmd:40]: FIX THIS

File: settings/POLREF/galil/galil06.cmd [hunk 12]
```diff
@@ -0,0 +1,44 @@
...
+epicsEnvSet "HOME_MODE_1" "$(GALIL)/gmc/galil_Home_Dummy_Do_Nothing.gmc"
+epicsEnvSet "HOME_MODE_2" "$(GALIL)/gmc/galil_Home_Dummy_Do_Nothing.gmc"
+epicsEnvSet "HOME_MODE_3" "$(GALIL)/gmc/galil_Home_Dummy_Do_Nothing.gmc"
+epicsEnvSet "HOME_MODE_4" "$(GALIL)/gmc/galil_Home_Dummy_Do_Nothing.gmc"
+epicsEnvSet "HOME_MODE_5" "$(GALIL)/gmc/galil_Home_Dummy_Do_Nothing.gmc"
+epicsEnvSet "HOME_MODE_6" "$(GALIL)/gmc/galil_Home_Dummy_Do_Nothing.gmc"
+epicsEnvSet "HOME_MODE_7" "$(GALIL)/gmc/galil_Home_Dummy_Do_Nothing.gmc"
+epicsEnvSet "HOME_MODE_8" "$(GALIL)/gmc/galil_Home_Dummy_Do_Nothing.gmc"
+epicsEnvSet "HOME_FOOTER" "$(GALIL)/gmc/galil_Default_Footer.gmc"
+epicsEnvSet "POLREF_SPECIAL" "C:/Instrument/Settings/config/NDXPOLREF/configurations/galil/combined_header.gmc"
```
Comment (Tom-Willemsen) [inline settings/POLREF/galil/galil06.cmd:40]: FIX THIS

**Review by Chsudeepta — APPROVED** [review]
The changes look fine. I have verified the following instruments: ALF, CRISP, IMAT, INTER, GEM, PEARL, SANDALS, SANS2D, SURF, TOSCA and WISH

**Pull request merged**